### PR TITLE
Update readme & added new issue template

### PR DESCRIPTION
**Pull request by Chris-Schnaufer**
## Description
Describe your changes, for example:
* Updated README.md from TERRA REF project
* Added issue template for submitting documentation requests

## Types of changes
* Changed README to be specific to AgPipeline
* New issue template

This is also tied to the documentation web site at: https://github.com/AgPipeline/AgPipeline.github.io/tree/transformers

back-reference to issue: https://app.zenhub.com/workspaces/ua-ag-data-science-5a57a3198339f11ba1c85775/issues/agpipeline/computing-pipeline/1

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/documentation_request.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+**Documentation Request Type**
+<!-- if there's more than one request type, please consider adding multiple tickets -->
+Please choose one of the following:
+- [ ] New documentation
+- [ ] Documentation needs enhancement or clarification
+- [ ] Documentation is broken, inaccurate, or out of date
+
+**What needs to be done**
+Please let us know what the question, problem, or need is
```
Comment (dlebauer) [inline .github/ISSUE_TEMPLATE/documentation_request.md]:
```suggestion
Please let us know your question, problem, or need. 
What do you want to do, and what have you tried so far? 
What specific parts of the documentation need to be clarified or extended?
```

File: .github/ISSUE_TEMPLATE/documentation_request.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+**Result**
+What's the result of doing this documentation task
```
Comment (dlebauer) [inline .github/ISSUE_TEMPLATE/documentation_request.md]:
```suggestion
What should this documentation task accomplish?
What should the user be able to do or understand after reading this documentation?
```

File: README.md [hunk 3]
```diff
@@ -2,15 +2,18 @@
 
 The computing pipeline is a data storage and computing system that provides researchers with access to all of the ‘raw’ data and derived plant phenotypes.
 
-This repository is the central issue repository for the AgPipeline organization.
+This repository is the central issue repository, and technical documentation landing site, for the AgPipeline organization.
```
Comment (dlebauer) [inline README.md]: Is the first paragraph (line 3) an accurate description / introduction to the AgPipeline?

File: .github/ISSUE_TEMPLATE/documentation_request.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+name: Documentation
+about: Ticket for when documentation needs to be changed, updated, or added
+title: Title of the documentation task
+labels: documentation
+assignees: ''
+
+---
+
+**Documentation Request Type**
+<!-- if there's more than one request type, please consider adding multiple tickets -->
```
Comment (MagicMilly) [inline .github/ISSUE_TEMPLATE/documentation_request.md]: I'm just going to comment on this for now until I know that I can suggest a commit without breaking everything. Suggestion: `For multiple requests, please submit one ticket per issue.`

File: README.md [hunk 5]
```diff
@@ -2,15 +2,18 @@
...
+If you are looking for a good place for overall project information, please look at our [Organization-info](https://github.com/AgPipeline/Organization-info) repo.
```
Comment (MagicMilly) [inline README.md]: Maybe `Please see our [Organization repo](https://github.com/AgPipeline/Organization-info) for overall project information.`

File: .github/ISSUE_TEMPLATE/documentation_request.md (hunk 2 above)
Comment (Chris-Schnaufer) [reply .github/ISSUE_TEMPLATE/documentation_request.md]: Accepted changes

File: .github/ISSUE_TEMPLATE/documentation_request.md (hunk 4 above)
Comment (Chris-Schnaufer) [reply .github/ISSUE_TEMPLATE/documentation_request.md]: done

File: README.md (hunk 5 above)
Comment (Chris-Schnaufer) [reply README.md]: Done

File: README.md (hunk 3 above)
Comment (Chris-Schnaufer) [reply README.md]: @dlebauer What do you think of the following for the first paragraph:
"This computing pipeline is a system for processing data to produce derived plant phenotypes."

Comment (dlebauer) [reply README.md]: "remote sensing data and derived plant phenotypes".

Comment (Chris-Schnaufer) [reply README.md]: Done

**Pull request merged**